pyvista/core/utilities/arrays.py

```diff
@@ -866,6 +866,15 @@ def cast_to_ndarray(arr, /, *, as_any=True, dtype=None, copy=False):
     """
     if as_any and not copy and dtype is None and isinstance(arr, np.ndarray):
         return arr
+
+    # needed to support numpy <1.25
```

Comment:
```suggestion
    # needed to support numpy <1.24 and vtk 9.0.3
```
See:
https://github.com/pyvista/pyvista/blob/e84354e72acf6d69277954cdc0f16c739aab6897/.github/workflows/testing-and-deployment.yml#L186-L188

Reply:
This is true for this vtk version.  If someone tries to use numpy 1.24 however with a newer version, it also will fail.  It just happens we only test <1.24

Reply:
OK. Thanks!